Convert.go

```diff
@@ -77,6 +77,12 @@ func Inverse(src EPSGCode, input []float64) ([]float64, error) {
 	return conv.inverse(input)
 }
 
+// CustomProjection provides write-only access to the internal projection list
+// so that projections may be added without having to modify the library code.
+func CustomProjection(code EPSGCode, str string) {
+	projStrings[code] = str
```

Comment:
I few ideas on this method:

* Implement a mutex since you're modifying a map
* Perform a check that the code is not already registered. I'm not sure if this should overwrite a code that already exists.